mailbox/event/json/src/main/scala/org/apache/james/event/json/DTOs.scala

```diff
@@ -75,15 +75,13 @@ object DTOs {
 
     def toJavaFlags(serializedFlags: Array[String]): JavaMailFlags = {
       serializedFlags
-        .map(toFlagBuilder)
-        .reduceOption(_ merge _)
-        .getOrElse(new FlagsBuilder())
+        .foldLeft(new FlagsBuilder)(_ add toJavaMailFlag(_))
```

Comment:
please avoid infix notation, AFAIK it generates compiler warning and in this particular case, it's harder to read IMO

Reply:
noticed, BTW that code is removed.